content/creator/wearables-and-emotes/wearables/linked-wearables.md

```diff
@@ -498,10 +499,18 @@ For programmatic collections, not all items have to be curated individually. The
 
 # Building the API
 
-In order for Linked Wearables to work, the third parties need to provide an API that will be queried by the Decentraland services (Catalyst) with 2 endpoints:
+In order for Linked Wearables to work, the third parties need to provide an API that will be queried by the Decentraland services (Catalyst) with 3 endpoints:
 
 1. `@GET /registry/:registry-id/address/:address/assets` - Retrieves a list of assets associated with a given address
 2. `@GET /registry/:registry-id/address/:address/assets/:id` - Validates if a DCL asset is owned by a user.
+3. `@POST /registry/:registry-id/ownership`that receives an array of data and performs validations in batch. 
+
+It's important to note that these endpoints are used to validate a user profile and what they are wearing, therefore there are performance SLA requirements to avoid harming the platform experience.  
+
+Considering that the client validates user profiles in batches so any delay in processing a single profile with LinkedWearables will impact the entire batch of users. This means that if the API performs slowly, it will negatively affect the overall performance of all users within that batch.
+
+To avoid damaging the platform performance all these endpoints must have an average response below the 500ms. This benchmark is crucial to prevent any degradation in the platform's overall performance.
```

Comment:
```suggestion
To avoid damaging the platform performance all these endpoints **must have an average response below the 500ms**. This benchmark is crucial to prevent any degradation in the platform's overall performance.
```